### PR TITLE
Add minified bundle (fixes #11)

### LINKED ISSUE #11: Add JS minification

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: config/webpack.js [hunk 1]
```diff
@@ -12,6 +16,17 @@ module.exports = {
     },
     failOnError: true, // don't report error to grunt if webpack find errors
     watch: true, // use webpacks watcher
-    //keepalive: true, // don't finish the grunt task
   },
+  prod: {
+    devtool: 'source-map',
```
Comment (kumar303) [inline config/webpack.js]: is there a way to move this up one level so it's inherited by both?

File: config/webpack.js [hunk 2]
```diff
@@ -12,6 +16,17 @@ module.exports = {
...
+    failOnError: true, // don't report error to grunt if webpack find errors
+    output: {
+      path: path.join(__dirname, '../public/dist/'),
+      filename: 'bundle.min.js',
```
Comment (kumar303) [inline config/webpack.js]: Just like for css, we probably need a way to link to this in prod, right? https://github.com/mozilla/payments-ui/blob/master/public/index.html#L18

File: config/webpack.js [hunk 3]
```diff
@@ -12,6 +16,17 @@ module.exports = {
...
+    },
+    plugins: [
+      new webpack.optimize.UglifyJsPlugin({minimize: true})
+    ],
+    watch: true, // use webpacks watcher
```
Comment (kumar303) [inline config/webpack.js]: same here - maybe there is a way to inherit this as a default for both configs?

**Comment by kumar303** [comment]
r+wc

File: config/webpack.js (hunk 2 above)
Comment (muffinresearch) [inline config/webpack.js]: Yeah, that'll be something needing to be done when we generate the html files (e.g. for l10n too)

File: config/webpack.js (hunk 3 above)
Comment (muffinresearch) [inline config/webpack.js]: This one probably not because tests inherit this.

File: config/webpack.js (hunk 1 above)
Comment (muffinresearch) [inline config/webpack.js]: Yep I reckon.

**Pull request merged**